src/redis-benchmark.c

```diff
@@ -427,6 +427,7 @@ static void setClusterKeyHashTag(client c) {
      * updateClusterSlotsConfiguration won't actually do anything, since
      * the updated_slots_count array will be already NULL. */
     if (is_updating_slots) updateClusterSlotsConfiguration();
+    node->current_slot_index = rand() % node->slots_count;
```

Comment:
`current_slot_index` in clusterNode is set to zero and never used outside this method. Can we remove this field?

Reply:
I was thinking the same.

Reply:
Yeah, this code was all written by atrix who didn't always write the best code. I like the idea of just removing it from the node here.

Reply:
It's a breaking change, but we can do that in Redis 8. I'm okay with that breaking change.

Reply:
OK, I added it to the Redis 8 project now.